net/wg-installer/Makefile

```diff
@@ -40,6 +41,16 @@ define Package/wg-installer-server/install
 	$(INSTALL_CONF) ./wg-server/config/wgserver.conf $(1)/etc/config/wgserver
 endef
 
+define Package/wg-installer-server-hotplug-babeld
+	$(call Package/wg-installer-server)
+	DEPENDS:=wg-installer-server
+endef
+
+define Package/wg-installer-server-hotplug-babeld/install
+	$(INSTALL_DIR) $(1)/etc/config/hotplug.d/
+	$(INSTALL_BIN) ./wg-server/hotplug.d/99-mesh-babeld $(1)/etc/config/hotplug.d/99-mesh-babeld
```

Comment:
This path looks wrong

Reply:
Yep.  Thanks. I'm still in WIP-Phase. ;)